File: Difficulty: Basic/Replace all 0's with 5/replace-all-0s-with-5.js
//{ Driver Code Starts
// Initial Template for javascript

'use strict';

process.stdin.resume();
process.stdin.setEncoding('utf-8');

let inputString = '';
let currentLine = 0;

process.stdin.on('data', inputStdin => { inputString += inputStdin; });

process.stdin.on('end', _ => {
    inputString =
        inputString.trim().split('\n').map(string => { return string.trim(); });

    main();
});

function readLine() { return inputString[currentLine++]; }

function main() {
    let t = parseInt(readLine());
    for (let i = 0; i < t; i++) {
        let input_line = readLine().split(' ');
        let n = parseInt(input_line[0]);

        let obj = new Solution();

        let ans = obj.convertFive(n);
        console.log(ans);

        console.log("~");
    }
}
// } Driver Code Ends

// User function Template for JavaScript

/**
 * @param {number} num
 * @returns {number}
 */
class Solution {
    convertFive(num) {
        // Convert number to string, replace all '0' with '5', and parse it back to integer
        return parseInt(num.toString().replace(/0/g, '5'));
    }
}

File: Difficulty: Basic/Replace all 0's with 5/replace-all-0s-with-5.java
//{ Driver Code Starts
import java.util.*;
class Replace{
public static void main(String[] args){
	Scanner sc=new Scanner(System.in);
	int t=sc.nextInt();
	while(t-->0){
		int n=sc.nextInt();
		Solution g=new Solution();
		System.out.println(g.convertFive(n));
	
System.out.println("~");
}
}
}
// } Driver Code Ends

/* Complete the function below */
class Solution{
    /**
     * Function to replace all 0s in the number with 5s.
     * @param n - The input number.
     * @returns int - The number after replacing all 0s with 5s.
     */
    public static int convertFive(int n) {
        // Convert the integer to a string to handle individual digits
        String num = Integer.toString(n);

        // Replace all occurrences of '0' with '5'
        num = num.replace('0', '5');

        // Convert the string back to an integer
        n = Integer.parseInt(num);

        // Return the modified integer
        return n;
    }
}
